packages/tasks/src/model-libraries.ts

```diff
@@ -448,6 +448,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path_extension:"nemo" OR path:"model_config.yaml"`,
 	},
+	"node-llama-cpp": {
```

Comment:
Are there any examples of where this is used? I don't see any tagged repos here: https://huggingface.co/models?other=node-llama-cpp

Note: Whichever repo has the tag `node-llama-cpp` would have the snippet you defined.

Reply:
We should only have `node-llama-cpp` as a local app option, not a library, for consistency with the rest

Reply:
Yes! read this: https://github.com/huggingface/huggingface.js/pull/949#pullrequestreview-2350820812

Reply:
agree

Reply:
IMO let's first ship in local apps and we can always revisit later

Reply:
Alright. I removed the library code snippet